webserver/webserver/src/main/java/io/helidon/webserver/RequestRouting.java

```diff
@@ -353,7 +290,14 @@ public void next() {
                     RoutedResponse nextResponse = new RoutedResponse(response);
                     RoutedRequest nextRequest = new RoutedRequest(this, nextResponse, nextItem.path, errorHandlers);
                     LOGGER.finest(() -> "(reqID: " + requestId() + ") Routing next: " + nextItem.path);
-                    requestSpan.log(nextItem.handlerRoute.diagnosticEvent());
+                    Span span = span();
```

Comment:
The span was a field and now has to be looked-up from the context every time.
Is the performance impact negligible ?

Reply:
I hope so. Otherwise there is a problem, as we do not have the interceptor to correctly handle all the events in the new service...

Reply:
I checked the implementation and it is much better to use a qualifier for performance reasons (as the list registry is not very optimal - it iterates until it finds an object that is of the correct type).

Reply:
So I will refactor the code to have a qualified instance of SpanContext